src/adapters/axios-adapter.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017 The "mysteriumnetwork/mysterium-vpn" Authors.
+ * Copyright (C) 2019 The "mysteriumnetwork/js-tequilapi" Authors.
```

Comment:
Yeah, but WebStorm is not smart enough to keep the year the same - you have to do it manually if you want to update copyright (i.e. because project name is incorrect, `mysterium-vpn` in this case).

Updated this one.

Reply:
Regexp to detect copyright in comments: `Copyright \(C\) 201. The "MysteriumNetwork\/node" Authors\.`